plugins/pulp_rpm/plugins/importers/yum/importer.py

```diff
@@ -62,6 +69,8 @@ def sync_repo(self, repo, sync_conduit, call_config):
         :return: report of the details of the sync
         :rtype:  pulp.plugins.model.SyncReport
         """
+        repo = platform_models.Repository.objects.get(repo_id=transfer_repo.id)
```

Comment:
Same as elsewhere, we should check the ``transfer_repo.repo_obj`` to see if we already have this object before going to the DB for it.